Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -1,4 +1,5 @@
 ﻿#region Using directives
+

```

Comment:
Remove extra line spaces. I can also see some bellow in the file.

Reply:
I think it's okay, but if you could take the time to pull the branch and format using your IDE just to be sure? Thanks!